examples/green-house-migration/legacy/greenhouse/batch/custom_fields.py

```
@@ -0,0 +1,43 @@
+    def fetch(self):
+        return []
+
+    def run(self):
+        summary = []
+        errored_models = []
+
+        for _model in self.model_types:
+            try:
+                print("Fetching custom fields for {model}...")
+                _data = gh_get("custom_fields/{model}")
+                saveentity_data(model, data, subfolder="custom_fields")
```

Ruff (0.12.2)

Undefined name `model`

(F821)